src/model/users/user.entity.ts

```diff
@@ -0,0 +1,41 @@
+import { Column, Entity, Index, PrimaryGeneratedColumn, Unique } from 'typeorm';
+
+@Entity({ name: 'user_v1' })
+@Unique(['email'])
+export class User {
```

Comment:
Have the new field for `user_uuid`. Use `uuid` instead of `integer id` as the real unique user id.

chatgbt says `import { v4 as uuidv4 } from 'uuid';` is the good option

Reply:
finish UUID as user_id type